variables.tf

```
@@ -37,3 +37,14 @@ variable "route_next_hop_type" {
 variable "route_next_hop_in_ip_address" {
   default = "10.10.1.1"
 }
+
+variable "apim_additional_routes" {
```

```suggestion
variable "additional_routes_apim" {
```

Keepsit same as what we do for app_gateway in aks module